sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -181,13 +181,22 @@ def display_data(self):
         'coder': self.coder.__class__.__name__
     }
 
+  def _get_hashable_destination(self, destination):
```

Comment:
I suggest to slightly rewrite this to avoid mixing `destination` and `key_destination`.
```
  def _extract_destination(self, table_reference):
    if isinstance(destination, bigquery_api.TableReference):
      return '%s:%s.%s' % (
          table_reference.projectId, table_reference.datasetId, table_reference.tableId)
    else:
       # Should we raise an error here? 

  def process(self, element, file_prefix):
    destination = _extract_destination(element[0])
    row = element[1]
```